.gitignore

```diff
@@ -23,3 +23,6 @@ src/mocks
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# IDE
+.idea
```

Comment:
Does this also work (be ignored) when you move this to your global Git config `~/.gitignore`?

Reply:
It does! Did not know about that feature!
Added .idea to `~/.gitignore_global`

Reply:
Dropped the commit.